Tools/AP_Periph/Parameters.cpp

```diff
@@ -371,6 +371,11 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     // @Path: ../libraries/AP_Scripting/AP_Scripting.cpp
     GOBJECT(scripting, "SCR_", AP_Scripting),
 #endif
+
+#if HAL_CANFD_SUPPORTED
+    // can node FD Out mode
+    GSCALAR(can_fdmode,     "CAN_FDMODE", 0),
```

Comment:
even though parameters aren't used yet, I think it is a good idea to start adding param doc comments on new periph parameters so we can see what they do